common/main/mai.xml

```diff
@@ -92,7 +92,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</codePatterns>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters>[क {क्ष} ख-घ च-ज {ज्ञ} झ-डड़{डं} ढढ़ ण त {त्र} थ-न प-र ल व श {श्र} ष-ह ा ि ी \u0941 \u0942 \u0947 \u0948 ो ौ]</exemplarCharacters>
+		<exemplarCharacters>[\u093C \u0902 ः क {क्ष} ख-घ च-ज {ज्ञ} झ-डड़{डं} ढढ़ ण त {त्र} थ-न प-र ल व श {श्र} ष-ह ा ि ी \u0941 \u0942 \u0947 \u0948 ो ौ]</exemplarCharacters>
```

Comment:
I'm slightly curious: I guess maybe U+0903 doesn't need to be escaped?

Reply:
Right, you can check the Hindi exemplars for comparison. Typically if the character has advance width (even if it is combining) then the exemplars use the actual character rather than the hex escape.